### DISCUSSION TITLE
Roop and MTB

### DISCUSSION THREAD

**Discussion by Piscabo** [melMass/comfy_mtb#54] [General]
Roop still breaks MTB. I installed it again to check if the error was resolved, but I got the error again, and even with roop gone I still get this error, which broke my venv. 
![image](https://github.com/melMass/comfy_mtb/assets/123719842/32027b98-49e8-49d1-8007-6d2fad468478)
Can you look into it? This guy is dissing your nodes, I had an argument about roop that is why I tried installing it again:
![image](https://github.com/melMass/comfy_mtb/assets/123719842/17f8a8fa-bd96-4e96-b1ff-8e26356c5d35)

**Comment by melMass** [comment]
Why do you need roop for that MTB does not do? I tried looking into but did not understand their approach it seems they bundle the A111 extension

**Reply by Piscabo** [reply]
Was trying to proof a point in your favor for MTB, against this 'flowwolf'. I am using MTB...

**Reply by melMass** [reply]
I did not take it otherwise but I'm trying to understand what they meant and if there is a reason for using Comfy-Roop over mtb, I will try it out again soon to see if I see a reason to remove these nodes from mtb, when I discovered that a dedicated extension existed I thought about doing it but my tests were not concluant

**Reply by melMass** [reply]
Errrf I read the thread on StableFoundation, I won't brother with that kind of attitude tbh, they most probably did wake up on the wrong side of bed
